test/integration/targets/azure_rm_virtualmachine/tasks/virtualmachine.yml

```diff
@@ -64,6 +65,35 @@
       public_ip_name: "{{ vm_name1 }}"
       security_group: "{{ vm_name1 }}"
 
+- name: Create virtual machine In Availability zone
+  azure_rm_virtualmachine:
+    admin_username: adminuser
+    admin_password: Password123!
+    resource_group: "{{ resource_group }}"
+    name: "{{ vm_name3 }}"
+    password: PASSWORD
+    vm_size: Standard_A0
+    availability_zone: 1
+    public_ip_allocation_method: Disabled
+    location: centralus
```

Comment:
From my understanding, the current integration tests are being ran on eastus region which doesn't support availability zones yet.

One of the solutions I see:

1. Create a new resource group in centralus region which happens to support availability zones and use it for the purpose of these tests
2. _Move_ the current resource group to centralus or any other region that supports Availability zone all along with other features being tested?

As I am new to this, I am open to suggestions regarding the best way to test this.

Reply:
I would like to move one of the `vm1` or `vm2` to `centralus` to avoid creating more VM, which requires many dependencies and costs a lot of time. 
BTW, the resource group is created out side of the integration test, and we run the whole yaml file in the resource group. My concern is whether the subscription, the integration test pays for, can allow us to create vm in `centralus`.

Reply:
Is there any way I can do that myself? or is it more in the hands of the people who are in charge of the Microsoft Azure account from which theses tests are ran (and which owns the resource group).

Reply:
@kamotos you should just add "location:" parameter in appropriate tasks and see if that works.

Reply:
@zikalino Sorry for the late answer. I think I have already added the `location` paramater to the integration test I wrote but that didn't work because the resource group is not in that region (location).

Or did you mean some other tasks when you were talking about "appropriate tasks"?